src/vmm/src/vmm_config/machine_config.rs

```diff
@@ -240,6 +247,8 @@ where
 /// to EC2 instances.
 #[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize, Versionize)]
 pub enum CpuFeaturesTemplate {
+    /// Custom template
+    CUSTOM,
```

Comment:
By adding anything to this enum field you break existing snapshots.
This is a problem I faced earlier with aarch64 templates.
This enum needs total overhaul. Maybe in separate PR.

Reply:
Could you give more details? Why does it break existing snapshots?

Reply:
I agree. I have split it out for versionization reasons as well.